mantidproject/mantid
# SANS Transmission - Support Added Event Data

*Requires  #31134 to be completed first*

**Description of work.**
Adds support for loading summed event data as the transmission (trans) run. We could already load raw trans data, but this was gated off.

**To test:**
- Open the ISIS SANS GUI
- Go to manage user directories
- Add system directories -> ISIS SANS -> ZOOM explicitly, it's not included by default
- Click user file and go to ISIS SANS -> ZOOM directory and pick any file available
- Go to sum runs
- Add 9946 twice, 9946 is raw event data
- Sum runs, you'll likely need to move the saved file into one of your managed user directories, as at least locally the save directory was not included
- Go back to the SANS GUI
- Enter 9898, 9946-add, 9946-add in the first 3 columns respectively (where 9946-add is the summed file output you just moved)
- Hit process, data should be produced

Fixes #27482 . 


---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
